styles/Editor.css

```diff
@@ -939,6 +939,47 @@
   font-size: 75%;
 }
 
+.ProseMirror .table-footer {
+  font-size: 14px;
+}
+
+.ProseMirror .table-footer > div:first-child {
+  padding-top: 8px;
+}
+
+.ProseMirror .table-footer > div:not(:first-child)::before {
+  content: '';
+  border-bottom: 1px dashed #C9C9C9;
+  display: block;
+  margin-bottom: 8px;
+}
+
+.ProseMirror .table-footer > div {
+  counter-reset: table_footnote;
+}
+
+.ProseMirror .table-footer > div p::before {
+  counter-increment: table_footnote;
+  content: counter(table_footnote);
+  display: inline-block;
+  position: relative;
+  cursor: pointer;
+  vertical-align: super;
+  font-size: 75%;
+  margin-right: 4px;
+}
+
+.ProseMirror .table-footer > div p {
```

Comment:
Do you think we can go a bit further here and add a class to that div and then do 
`.ProseMirror .table-footer .some-class p {...` ?

Reply:
Done!